<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculatrice</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="pile">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>171</width>
      <height>271</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>481</width>
      <height>278</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="3" column="0">
      <widget class="QPushButton" name="b0">
       <property name="text">
        <string>0</string>
       </property>
       <property name="shortcut">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="b7">
       <property name="text">
        <string>7</string>
       </property>
       <property name="shortcut">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="b8">
       <property name="text">
        <string>8</string>
       </property>
       <property name="shortcut">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="b9">
       <property name="text">
        <string>9</string>
       </property>
       <property name="shortcut">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="b4">
       <property name="text">
        <string>4</string>
       </property>
       <property name="shortcut">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="b5">
       <property name="text">
        <string>5</string>
       </property>
       <property name="shortcut">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="b6">
       <property name="text">
        <string>6</string>
       </property>
       <property name="shortcut">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="b1">
       <property name="text">
        <string>1</string>
       </property>
       <property name="shortcut">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="b2">
       <property name="text">
        <string>2</string>
       </property>
       <property name="shortcut">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="b3">
       <property name="text">
        <string>3</string>
       </property>
       <property name="shortcut">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="bVirgule">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>,</string>
       </property>
       <property name="shortcut">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="bDollar">
       <property name="text">
        <string>$</string>
       </property>
       <property name="shortcut">
        <string>$</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QPushButton" name="bCote">
       <property name="text">
        <string>'</string>
       </property>
       <property name="shortcut">
        <string>'</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QPushButton" name="bSpace">
       <property name="text">
        <string>SPACE</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="bSup">
       <property name="text">
        <string>&lt;--</string>
       </property>
       <property name="shortcut">
        <string>Backspace</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QPushButton" name="bEval">
       <property name="text">
        <string>EVAL</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+E</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="2">
      <widget class="QPushButton" name="bEnter">
       <property name="text">
        <string>=</string>
       </property>
       <property name="shortcut">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="bPlus">
       <property name="text">
        <string>+</string>
       </property>
       <property name="shortcut">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="bMoins">
       <property name="text">
        <string>-</string>
       </property>
       <property name="shortcut">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="bDivision">
       <property name="text">
        <string>/</string>
       </property>
       <property name="shortcut">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="bFois">
       <property name="text">
        <string>*</string>
       </property>
       <property name="shortcut">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QPushButton" name="bSign">
       <property name="text">
        <string>SIGN</string>
       </property>
      </widget>
     </item>
     <item row="14" column="4">
      <widget class="QPushButton" name="bInv">
       <property name="text">
        <string>INV</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="bMod">
       <property name="text">
        <string>MOD</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="bFact">
       <property name="text">
        <string>!</string>
       </property>
       <property name="shortcut">
        <string>!</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="bSqr">
       <property name="text">
        <string>SQR</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="bCube">
       <property name="text">
        <string>CUBE</string>
       </property>
      </widget>
     </item>
     <item row="12" column="6">
      <widget class="QPushButton" name="bPow">
       <property name="text">
        <string>POW</string>
       </property>
      </widget>
     </item>
     <item row="14" column="6">
      <widget class="QPushButton" name="bSqrt">
       <property name="text">
        <string>SQRT</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>521</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Op√©rations pile</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>491</width>
       <height>62</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QPushButton" name="bMean">
        <property name="text">
         <string>MEAN</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="bSum">
        <property name="text">
         <string>SUM</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="bDup">
        <property name="text">
         <string>DUP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="bSwap">
        <property name="text">
         <string>SWAP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bClear">
        <property name="text">
         <string>CLEAR</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+C</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="bDrop">
        <property name="text">
         <string>DROP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QPushButton" name="bCos">
       <property name="text">
        <string>COS</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="bSin">
       <property name="text">
        <string>SIN </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="bTan">
       <property name="text">
        <string>TAN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="bSinh">
       <property name="text">
        <string>SINH</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="bTanh">
       <property name="text">
        <string>TANH</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="bLn">
       <property name="text">
        <string>LN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="bLog">
       <property name="text">
        <string>LOG</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="bCosh">
       <property name="text">
        <string>COSH</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="title">
     <string>Nombre d'√©l√©ments</string>
    </property>
    <widget class="QLineEdit" name="nbOpPile">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Param√®tres</string>
    </property>
    <widget class="QCheckBox" name="Complexe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Complexe Autoris√©s</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>86</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="bEntier">
        <property name="text">
         <string>Entier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bRationnel">
        <property name="text">
         <string>Rationnel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bReel">
        <property name="text">
         <string>R√©el</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>16</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>101</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="bRadian">
        <property name="text">
         <string>Radian</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bDegre">
        <property name="text">
         <string>Degr√©</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRetablir"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAnnuler">
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRetablir">
   <property name="text">
    <string>R√©tablir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
